### PR TITLE
Prepare docs 1st wave (RC1) April 2024

### PR DESCRIPTION
**Pull request by eladkal**

### CODE REVIEW

File: airflow/providers/amazon/CHANGELOG.rst
Comment (vincbeck) [inline airflow/providers/amazon/CHANGELOG.rst]: It's been reverted

```suggestion
```

**Review by potiuk — APPROVED** [review]
Did not found anything unusuai.

File: airflow/providers/amazon/CHANGELOG.rst
Comment (eladkal) [reply airflow/providers/amazon/CHANGELOG.rst]: This entry will be moved to hidden from log when I refresh the docs before merge

**Review by vincbeck — APPROVED** [review]

File: airflow/providers/apache/hive/CHANGELOG.rst
Comment (Taragolis) [inline airflow/providers/apache/hive/CHANGELOG.rst]: This fix around deprecated code not relevant anymore after it removed into the #38859

Comment (Taragolis) [reply airflow/providers/apache/hive/CHANGELOG.rst]: So we could drop it from the changelog

**Review by hussein-awala — APPROVED** [review]

File: airflow/providers/amazon/CHANGELOG.rst
Comment (eladkal) [reply airflow/providers/amazon/CHANGELOG.rst]: https://github.com/apache/airflow/pull/38863/commits/df5ae10b80490d4cd382c9a028a393b10f136c4b

File: airflow/providers/apache/hive/CHANGELOG.rst
Comment (eladkal) [reply airflow/providers/apache/hive/CHANGELOG.rst]: https://github.com/apache/airflow/pull/38863/commits/8ab2633d5d1d629eec43b04c0800f3e0cca30969

**Review by Taragolis — APPROVED** [review]

**Comment by hussein-awala** [comment]
@eladkal I hope it is not too late to include https://github.com/apache/airflow/pull/38199, I had to bump min version to avoid implementing a complicated import logic.

**Review by amoghrajesh — APPROVED** [review]
LGTM +1 with respect to my content and other general content

**Comment by pankajkoti** [comment]
@eladkal just wish to check if it's easily possible to include https://github.com/apache/airflow/pull/38881 in this wave. If not, that's okay too. thanks!

**Comment by eladkal** [comment]
> @eladkal just wish to check if it's easily possible to include #38881 in this wave. If not, that's okay too. thanks!

yes, I am waiting for @hussein-awala PR to be merged then I will rebase this branch and close the release

**Review by hussein-awala — APPROVED** [review]

**Pull request merged**
